docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -396,11 +375,22 @@ endif::[]
 +
 <1> Select an existing storage class in your {OpenShiftShort} cluster.
 
+This example provides the required infrastructure database and messaging services for 1 Compute cell
+named `cell1`. Adjust the names, counts, IP addresses, and numbers, such as `replicas`, `storage`, or `storageRequest`, as needed.
+
 .Verification
 
-* Verify that MariaDB is running:
+* Verify that MariaDB and RabbitMQ are running for all defined cells:
 +
 ----
-$ oc get pod openstack-galera-0 -o jsonpath='{.status.phase}{"\n"}'
-$ oc get pod openstack-cell1-galera-0 -o jsonpath='{.status.phase}{"\n"}'
+$ RENAMED_CELLS="cell1"
```

Comment:
@gibizer this provides an example, how we can extend to multi-cell (as it is shown in follow up PR), yet it still deploys s single cell1 from default tripleo cell